runtimevar/driver/driver.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package driver provides the interface for providers of runtimevar.  This serves as a contract
-// of how the runtimevar API uses a provider implementation.
+// Package driver defines an interface that the runtimevar package uses to
```

Comment:
"an interface" in conceptual sense vs "interfaces" or "set of interfaces", referring to the Go interfaces - database/sql uses the latter, should we?

Reply:
Which `database/sql`? https://golang.org/pkg/database/sql/ says "Package sql provides a generic interface around SQL (or SQL-like) databases."

The wording in `driver` packages means `interface(s)` referring to Go interfaces, not in a conceptual sense. I used the singular here because I thought that this package only defined a single interface, but it actually defines 2, so I updated it to "a set of interfaces".

Reply:
I was referring to https://golang.org/pkg/database/sql/driver/ since this was a comment for driver, not the main top-level package.